Code/Editor/EditorFramework/IPC/EngineProcessConnection.cpp

```diff
@@ -288,7 +288,10 @@ bool ezEditorEngineProcessConnection::SendMessage(ezProcessMessage* pMessage)
 
 ezResult ezEditorEngineProcessConnection::WaitForMessage(const ezRTTI* pMessageType, ezTime timeout, ezProcessCommunicationChannel::WaitForMessageCallback* pCallback)
 {
-  EZ_PROFILE_SCOPE(pMessageType->GetTypeName());
+  // NOTE: When Tracy is enabled, there is no conversion for ezStringView to const char*, so this is a workaround.
+  ezStringBuilder sb;
+  pMessageType->GetTypeName().GetData(sb);
+  EZ_PROFILE_SCOPE(sb);
```

Comment:
This has to work without modification here, otherwise it is inconvenient and wastes performance for unnecessary string copies.

I'll mark up all cases that I see that need to be fixed.

Reply:
Ill make a converter for ez string types within Profiler_Tracy